cmd/statusd/utils.go

```diff
@@ -1488,3 +1492,19 @@ func testValidateNodeConfig(t *testing.T, config string, fn func(common.APIDetai
 
 	fn(resp)
 }
+
+func testProfiling(t *testing.T) bool {
+	StartProfiling(C.CString(""))
+
+	address, _, _, err := statusAPI.CreateAccount(TestConfig.Account1.Password)
+	if err != nil {
+		t.Errorf("could not create account: %v", err)
+		return false
+	}
+
+	t.Log("Account created: ", address)
+
+	profiling.Stop()
+
+	return true
+}
```

Comment:
It's better be tested without C bindings. Especially considering my other comments.